### ISSUE TITLE
Too many errors 

### ISSUE THREAD

**Issue by ipinky77** [sologenic/sologenic-expo-wallet#131]
![2406720E-6ECC-46CB-AB51-4DF762726088](https://user-images.githubusercontent.com/42062924/144369205-6e906fac-2540-475a-9f85-c426249df0bc.png)

trying to cancel an order I receive hundreds of errors. Making it literally impossible to continue using the app. A one time notification is sincerely enough but not repeatedly

**Comment by nduarte23** [comment]
This is beyond ridiculous. Why can I not convert my XRP into solo on the app. And now I can’t even withdraw it. Too many errors. Someone needs to fix this. I’ve sent my XRP to this app and I can’t get it out now. Need help ASAP!!!

**Comment by ledartistry** [comment]
Yeah mines been doing this shit all day!  I want to withdrawal just a little XRP? What’s up ?

**Comment by ledartistry** [comment]
I heard the XRPl was having problems is this related to that?

**Comment by ledartistry** [comment]
![A71BA794-E500-4F24-A1FE-30114F927D2D](https://user-images.githubusercontent.com/81092497/144377013-8879abcc-f364-44e8-a173-f6281e36e12d.jpeg)

**Comment by kamokat** [comment]
This is bad . Almost a whole day and still not fixed.  Imagine what its going to be like a snapshot. No will be able to sell or convert solo. Someone's site must be working because money coming in from somewhere.  
We are screwed if we continue to use solodex
